kuwala/pipelines/google-poi/resources/categories.json

```diff
@@ -0,0 +1,260 @@
+        "category": "food",
+        "tags": [
+            "banquet_hall",
+            "lunch_restaurant",
+            "restaurant"
+        ]
+    },
+    "groceries": {
+        "category": "groceries",
+        "tags": [
+            "grocery_store",
+            "supermarket"
+        ]
+    },
+    "medical": {
+        "category": "medical",
+        "tags": [
+            "hospital",
+            "medical_center"
+        ]
+    },
+    "misc": {
+        "category": "misc",
+        "tags": [
+            "attraction",
+            "bar",
+            "cafe",
+            "casino",
+            "church",
+            "concert_hall",
+            "conference_center",
+            "convenience_store",
+            "convention_center",
+            "cultural_center",
+            "dome",
+            "event_venue",
+            "farm",
+            "flooring_store",
+            "garden_center",
+            "golf_course",
+            "government_office",
+            "hardware_store",
+            "hospital",
+            "immigration_and_naturalization_service",
+            "library",
+            "machines",
+            "museum",
+            "outdoor_furniture_store",
+            "paint_store",
+            "park",
+            "plant_nursery",
+            "plumbing_supply_store",
+            "police_department",
+            "police_station",
+            "produce_market",
+            "pub",
+            "public_library",
+            "restaurant",
+            "school",
+            "stadium",
+            "store",
+            "trailer_rental_service",
+            "train_station",
+            "variety_store",
+            "wedding_venue",
+            "woods"
+        ]
+    },
+    "office": {
+        "category": "office",
+        "tags": [
+            "business_center",
+            "corporate_office",
+            "government_office",
+            "police_department"
+        ]
+    },
+    "publicService": {
+        "category": "publicService",
+        "tags": [
+            "association_or_organization",
+            "city_hall",
+            "immigration_and_naturalization_service"
+        ]
+    },
+    "publicTransportation": {
+        "category": "publicTransportation",
+        "tags": [
+            "pub",
+            "train_station"
+        ]
+    },
+    "recreation": {
+        "category": "recreation",
+        "tags": [
+            "botanical_garden",
+            "golf_club",
+            "horse_boarding_stable",
+            "park",
+            "woods"
+        ]
+    },
+    "religiousBuilding": {
+        "category": "religiousBuilding",
+        "tags": [
+            "cathedral",
+            "church",
+            "mosque"
+        ]
+    },
+    "service": {
+        "category": "service",
+        "tags": [
+            "city_hall",
+            "government_office",
+            "immigration_and_naturalization_service",
+            "police_department",
+            "public_library",
```

Comment:
They are all already map to different categories where they also fit better. "Service" is for private businesses such as a tax consultant or lawyer.

Reply:
i have currently trailer_rental_service here for service is it a problem?

Reply:
I'd say it's fine